### ISSUE TITLE
[MAC:FR]Health Data Pop Up for french is not appearing as that in English

### ISSUE THREAD

**Issue by rawat11** [adobe/brackets#11387]
Steps:

1 Launch Brackets

Result

Health Data first pop up screen does not appear as that in ENG

![screen shot 2015-07-13 at 04 09 31](https://cloud.githubusercontent.com/assets/10069885/8648415/5839ede2-2915-11e5-9695-646a1760dfd1.png)

Eng SS

![screen shot 2015-07-13 at 04 13 17](https://cloud.githubusercontent.com/assets/10069885/8648448/8cd49fde-2915-11e5-9107-422d7271b964.png)

**Comment by rawat11** [comment]
Japanese look different as well

![screen shot 2015-07-14 at 2 27 22](https://cloud.githubusercontent.com/assets/10069885/8670216/e8102d5e-29cf-11e5-9938-adac9c89f8f4.png)

**Comment by nethip** [comment]
@rawat11 I fixed this exact bug last release and it somehow crept in again. It is to do with the string change. We have some HTML formatting in the string, and the strings are getting replaced with new set of strings.

**Comment by rawat11** [comment]
@nethip Yes, there is some formatting issue it wasn't coming before, now is reproducible on both win and mac.
